test/FieldTests/HintTest.java

```diff
@@ -0,0 +1,15 @@
+package FieldTests;
+
+import Field.*;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class HintTest {
+
+    /*@Test
```

Comment:
Don't commit commented out code unless you've got a really good reason. Happy to go through some advanced git things like stashing or interactive add next time we pair.

Reply:
That would be awesome to know thanks! What if you have commented code you don't want to get rid of but you need to commit?